xml/System/String.xml

```diff
@@ -563,8 +566,8 @@ You can download the [Sorting Weight Tables](https://www.microsoft.com/download/
 
  ]]></format>
     </remarks>
-    <related type="ExternalDocumentation" href="https://www.microsoft.com/download/details.aspx?id=10921">Sorting Weight Tables for Windows operating systems</related>
-    <related type="ExternalDocumentation" href="https://www.unicode.org/Public/UCA/latest/allkeys.txt">Default Unicode Collation Element Table, for Linux and macOS</related>
+    <related type="ExternalDocumentation" href="https://www.microsoft.com/download/details.aspx?id=10921">Sorting Weight Tables, for Windows operating systems (.NET Framework and .NET Core only)</related>
+    <related type="ExternalDocumentation" href="https://www.unicode.org/Public/UCA/latest/allkeys.txt">Default Unicode Collation Element Table, for .NET 5+ on Linux, macOS, or Windows 19H1 or later, and .NET Core on Linux and macOS</related>
```

Comment:
Same note about `Windows 19H1` -> `Windows 10 May 2019 Update`.

Reply:
This link text was too long so I just ended up shortening them both.